gui/wxpython/gmodeler/toolbars.py

```diff
@@ -29,15 +29,15 @@ class ModelerToolbar(BaseToolbar):
     def __init__(self, parent):
         BaseToolbar.__init__(self, parent)
 
+        # realize the toolbar
+        self.Realize()
+
         # workaround for http://trac.wxwidgets.org/ticket/13888
         if sys.platform == "darwin":
```

Comment:
This workaround, is what causes the problem. It is apparently not needed anymore, can't tell since when...

Reply:
That is the workaround in our code: `if sys.platform == "darwin": ...`

Reply:
`if sys.platform == "darwin" and not CheckWxVersion([4, 2, 1]):` seems to work for me.

Reply:
@nilason: What about `if sys.platform == "darwin" and not CheckWxVersion([4, 2, 0]):`? If the 4.2 version of wx brought the incosistency to macOS, it was probably already in the 4.2.0 version.

Reply:
> What about if sys.platform == "darwin" and not CheckWxVersion([4, 2, 0]):?

I could test only with 4.2.1 and 4.2.2.

>  If the 4.2 version of wx brought the incosistency to macOS, it was probably already in the 4.2.0 version.

Let us go safe and fix what has been tested. If someone, however unlikely, will attempt to run with 4.2.0 and report then I'd say we're happy to address it.

Reply:
Fixed in your proposed way in 289dd1ce95a88ab138bc855e33336a75481043e6

Reply:
You will need to add something like `from core.globalvar import CheckWxVersion`.